DEVELOPMENT.md

```diff
@@ -2,6 +2,33 @@
 
 This document intends to establish guidelines which build a transparent, open mechanism for deciding how to evolve the OpenAPI Specification. The OpenAPI Technical Steering Committee (TSC) will initially follow these processes when merging changes from external contributors or from the TSC itself. This guideline document will be adjusted as practicality dictates.
 
+### Essential Policies
+
+If in doubt about a policy, please [ask on our Slack](https://communityinviter.com/apps/open-api/openapi) before opening a PR.
+
+_Note that the later sections of this document have not been updated recently.  Until that has happened, if this section and later sections disagree, this section is to be considered the current policy._
```

Comment:
I'd discourage adding time-sensitive content into a file we will probably continue to fail to update. Directing people to slack is the right thing in almost every case of confusion!

Reply:
@lornajane yeah... I'm not sure what to do here.  This file is huge, and a _lot_ of it needs rewriting.  I'd almost rather delete everything that's not current.  Better to have undefined processes that documentation that is, at this point, outright wrong.